app/views/katello/api/v2/content_view_environments/show.json.rabl

```diff
@@ -3,6 +3,7 @@ extends 'katello/api/v2/common/identifier'
 
 extends 'katello/api/v2/common/timestamps'
 attributes :default_environment? => :default
+attributes :id
```

Comment:
This doesn't show up in `GET /api/hosts/:host_id` (but that's where I need it ;) )

Reply:
```patch
--- app/views/katello/api/v2/content_facet/base.json.rabl
+++ app/views/katello/api/v2/content_facet/base.json.rabl
@@ -31,6 +31,9 @@ child :content_view_environments => :content_view_environments do
   node :label do |cve|
     cve.label
   end
+  node :id do |cve|
+    cve.id
+  end
 end
 
 attributes :content_view_environment_labels
```